base/repl/LineEdit.jl

```diff
@@ -970,15 +967,15 @@ function write_response_buffer(s::PromptState, data)
 end
 
 mutable struct SearchState <: ModeState
-    terminal
-    histprompt
+    terminal::AbstractTerminal
+    histprompt # ::HistoryPrompt
```

Comment:
Why comment the type annotation here? (similar question in other places, e.g. with `CompletionProvider`)